ru_RU.ini

```diff
@@ -156,6 +156,7 @@ Delete completed = Удаление завершено
 DeleteConfirm = Эти данные будут удалены.\nХотите продолжить?
 Deleting = Удаление\nПожалуйста, подождите...
 Load = Загрузить
+Load State = Загрузить
```

Comment:
You shouldn't translate that because the pop-up dialog is not finished yet.
This string should be translated in [Pause], not in [Dialog]...
I have already discussed about that with @hrydgard : https://github.com/hrydgard/ppsspp/commit/a125178b0bd422bac040b1a7435b1c62f0c9537e#commitcomment-9138121